packages/core/src/Stores/client-store.js

```diff
@@ -1652,15 +1652,21 @@ export default class ClientStore extends BaseStore {
                     statement: 1,
                 })
             );
-            const account_settings = (await WS.authorized.cache.getSettings()).get_settings;
+
+            const account_settings =
+                Object.keys(account_settings).length === 0
+                    ? (await WS.authorized.getSettings()).get_settings
+                    : this.account_settings;
+
             if (account_settings) this.setPreferredLanguage(account_settings.preferred_language);
             await this.fetchResidenceList();
             await this.getTwoFAStatus();
             if (account_settings && !account_settings.residence) {
                 this.root_store.ui.toggleSetResidenceModal(true);
             }
-
-            await WS.authorized.cache.landingCompany(this.residence).then(this.responseLandingCompany);
+            await WS.authorized.cache
+                .landingCompany(this.residence || account_settings.country_code)
```

Comment:
<img width="486" alt="image" src="https://user-images.githubusercontent.com/56330681/225530492-3df9ddf8-00a2-4ddf-8eac-4a96aa5d9471.png">
`this.residence` is same as `account_settings.country_code`. Why do we need this change?

Reply:
Alright @balakrishna-deriv, I'll update it